config/checkstyle/checkstyle-suppressions.xml

```diff
@@ -23,7 +23,7 @@
     <suppress checks="ParameterNumber" files="UserDataBuilder.java|ServiceEndpointCollector.java"/>
     <suppress checks="EmptyLineSeparator" files=".*Description.java|.*Descriptions.java|Notes.java"/>
     <suppress checks=".*" files="PrometheusResponse.java"/>
-    <suppress checks="CyclomaticComplexity" files="PrometheusEvaluator.java|MetadataSetupService.java|AzureRoleManager.java|DBStackToDatabaseStackConverter.java"/>
+    <suppress checks="CyclomaticComplexity" files="PrometheusEvaluator.java|MetadataSetupService.java|AzureRoleManager.java|DBStackToDatabaseStackConverter.java|ClusterServiceConfig.java"/>
```

Comment:
please don't add exception if not really necessary. usually CyclomaticComplexity can be workaround with some refactor. Or if it's only one method, you can use `@SuppressWarnings` annotation, so the rest of the class would be still validated

Reply:
Good point. This warning is coming from the `equals()` method which I don't think should be refactored. 

Just tried using `@SuppressWarnings` but I couldn't get it working with this checkstyle config.

Using `//CHECKSTYLE:OFF: CyclomaticComplexity` however worked.